### DISCUSSION TITLE
[v4] Normal font weight is not respected

### DISCUSSION THREAD

**Discussion by entity** [tailwindlabs/tailwindcss#14656] [Help] [answered]
**What version of Tailwind CSS are you using?**

^4.0.0-alpha.26

**What build tool (or framework if it abstracts the build tool) are you using?**

Vite (v5.4.8) with React (v18.3.1)

**What version of Node.js are you using?**

v21.7.3

**What browser are you using?**

Safari

**What operating system are you using?**

macOS

**Reproduction URL**

https://play.tailwindcss.com/jFiizARA8r

**Describe your issue**

When setting a font weight via the new CSS variable syntax:
```css
@theme {
  --font-weight-normal: 375;
}
```

It's not set globally by default for all html tags, and it also shows as "NaN" within Safari dev tools. I have tried changing it to `'375'` and the same happens.

![CleanShot 2024-10-12 at 7  46 32@2x](https://github.com/user-attachments/assets/93217f4b-b7f4-4e95-8e6c-19bfe741cf13)

**Comment by adamwathan** [comment, marked as answer]
Hey! This looks like it's working as expected to me, the `font-normal` class is correctly setting the font weight to 375:

<img width="983" alt="image" src="https://github.com/user-attachments/assets/5c82e83a-698d-40bc-aa23-53729ff2f79e">


If you're expecting it to be set for the whole document, you need to add the `font-normal` class to the `<html>` or `<body>` elements — we don't automatically take whatever you set for `font-normal` and apply it to the document. Same behavior in v3 👍

**Reply by entity** [reply]
Hey :) Thanks for such a quick reply, my bad I thought it would be set globally, I didn't realise I needed to set it in my html/body.

Loving v4 so far, great job with everything! Congrats on the house

**Reply by entity** [reply]
Also, is there any reason why Safari reports the font weight as "NaN" by any chance?
